lms/djangoapps/courseware/date_summary.py

```diff
@@ -0,0 +1,250 @@
+"""
+This module provides date summary blocks for the Course Info
+page. Each block gives information about a particular
+course-run-specific date which will be displayed to the user.
+"""
+import pytz
+
+from datetime import datetime
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from edxmako.shortcuts import render_to_string
+from lazy import lazy
+
+from course_modes.models import CourseMode
+from verify_student.models import VerificationDeadline, SoftwareSecurePhotoVerification
+from student.models import CourseEnrollment
+
+
+# pylint: disable=missing-docstring
```

Comment:
Seriously!? I thought it was smart enough to pull from the base class. Oh well. Can you move this to the top of the file?